### DISCUSSION TITLE
Is there a way to disable the injected keymaps by other plugins?

### DISCUSSION THREAD

**Discussion by ZachVec** [LazyVim/LazyVim#4542] [Q&A] [answered]
I wonder if there's a way to disable the `<leader>ss` keymap for Telescope, which was [injected by Aerial](https://www.lazyvim.org/keymaps#telescopenvim-1).

I've replaced all of the Telescope keymaps with my own, but this one persists, even after trying `{ "<leader>ss", false }`.

For now, I'm just hiding it in which-key.

**Comment by dpetka2001** [comment, marked as answer]
You disable this key in the Telescope plugin spec like the others that you did and define a new one.

**Reply by ZachVec** [reply]
![image](https://github.com/user-attachments/assets/88026695-b57d-47cd-af64-461c0f267948)

I have disabled all of the Telescope keymaps, but this one persists.

**Reply by dpetka2001** [reply]
You're doing something wrong in your configuration. I just tried it and it works correctly.

**Reply by ZachVec** [reply]
Yes, you are right. I accidentally added aerial to the Lazy's import earlier.

![image](https://github.com/user-attachments/assets/58fbf7df-e6d7-4684-959c-5038738a5938)
